build/Targets/VSL.Versions.targets

```diff
@@ -73,7 +73,7 @@
   </Choose>
 
   <!-- Returns the current build version. Used in .vsixmanifests to substitute our build version into them -->
-  <Target Name="GetBuildVersion" Outputs="$(BuildVersion)" />
+  <Target Name="GetBuildVersion" Outputs="$(RoslynSemanticVersion).$(BuildNumberPart1.Trim())$(BuildNumberPart2.Trim())" />
```

Comment:
As for the Trim(), it's probably some legacy support for the old signed-build system/TFS and it may or may not be needed.